recipe/bld.bat

```diff
@@ -1,8 +1,14 @@
 mkdir "%SRC_DIR%"\build
 pushd "%SRC_DIR%"\build
 
-if NOT "%cuda_compiler_version%"=="None" (
+if not "%cuda_compiler_version%" == "None" (
     set "CMAKE_ARGS=-DUSE_CUDA=ON %CMAKE_ARGS%"
+
+    :: Workaround CUB build errors on CUDA 11.8
+    if not "%cuda_compiler_version%" == "11.8" (
+        set "CXXFLAGS=%CXXFLAGS% -DWIN32_LEAN_AND_MEAN"
+        set "NVCC_APPEND_FLAGS=%NVCC_APPEND_FLAGS% -DWIN32_LEAN_AND_MEAN"
+    )
```

Comment:
Think we need something like this for Windows CUDA 11.8. That or we use a newer version of CUB than the CTK includes

xref: https://github.com/facebookresearch/pytorch3d/issues/1614#issuecomment-1719514892